README.md

```diff
@@ -59,6 +59,11 @@ These templates can generate savings estimates for your environment.
 - [Google Cloud SQL Idle Instance Recommender](./cost/google/cloud_sql_idle_instance_recommendations)
 - [Google Idle VM Recommender](./cost/google/idle_vm_recommendations)
 
+#### Kubecost
+
+- [Kubecost Request Rightsizing Recommendations](./cost/kubecost/sizing)
+- [Kubecost Cluster Rightsizing Recommendations](./cost/kubecost/cluster)
+
```

Comment:
This table of contents gets generated by a script -- we no longer need to manually manage this list.  The placement is determined by the PT metadata.

https://github.com/flexera-public/policy_templates/blob/master/.github/workflows/generate-readme-policy-table-of-contents.yaml